asv/plugins/virtualenv.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, conf, python, requirements, tagged_env_vars):
                                          tagged_env_vars)
 
         try:
-            import virtualenv # noqa F401 unused, but required in case the previous step does not work
+            import virtualenv # noqa F401 unused, but required to manage an environment
```

Comment:
This is not required to manage any environment. This import is just used to test whether virtualenv is installed or not, and its goal is to fail here with a useful error message (instead of elsewhere I guess) if the package is not installed.

Reply:
Great, thank you so much for the explanation!